README.md

```diff
@@ -1359,13 +1358,26 @@ var scope = nock('http://api.myservice.com')
 
 # Contributing
 
-## Generate Changelog
+## Commit Message conventions
 
-This should be done immediately after publishing a new version to npm.
+`nock` releases are automated using [semantic-release](https://github.com/semantic-release/semantic-release).
+To automatically calculate the correct version number as well as changelogs,
+three commit message conventions need to be followed
 
-```
-$ npm run changelog
-```
+- Commit bug fixes with `fix: ...` or `fix(scope): ...` prefix
+- Commit new features with `feat: ...` or `feat(scope): ...` prefix
```

Comment:
What would be an example of a `scope` for this project?

Reply:
An example would be `nockback` or `recorder`, different modules. I also use the scope for chores like `chore(gitignore): new-ingored-file` or docs like `docs(README): add section xyz`